### ISSUE TITLE
Linux Build

### ISSUE THREAD

**Issue by ryanleesipes** [gavanw/vqisosmall#12]
Considering it is open source, I think you could capture more contributors by supporting a Linux build.

**Comment by gavanw** [comment]
Yes, there are people working on Linux porting (at least, at some rate). I'll look into a few more issues myself when I get time (I noticed some recent commits kind of messed things up).

**Comment by ryanleesipes** [comment]
@gavanw I am very pleased with this answer :smile:

**Comment by ristew** [comment]
I got it to run on linux with the following steps
1. clone the repo, extract resources
2. put the lzz binary from http://www.lazycplusplus.com/download.html into the main directory
3. remove the VoxelQuest folder
4. `cmake .`
5. `make`
6. create a folder (I call it "game" but it doesn't matter) and put the VoxelQuest executable there
7. move into the folder, and play the game!

**Comment by OKNoah** [comment]
@ristew's steps worked for me on macOS with these additional steps before doing `cmake .`

This requires [homebrew](https://brew.sh) be installed.

```
brew install cmake
brew install GLEW
brew install SFML
```

It also seems necessary to run the game from the terminal `./VoxelQuest` rather than double-click it. And don't forget to get the `data` directory from this `.rar`: http://gavanw.itch.io/vqiso

**Comment by gavanw** [comment]
Cool, thanks for the info! The terminal run thing might be due to Apple's
weird app-packaging, could be wrong though.
